app/models/calendar_event.rb

```diff
@@ -84,9 +84,15 @@ def self.project_date_filter(range, scope = Project)
                 {first: range.first, last: range.last})
   end
 
-  def self.project_step_date_filter(range, scope = ProjectStep)
-    scope.where("actual_end_date BETWEEN :first AND :last OR scheduled_start_date BETWEEN :first AND :last "\
-      "OR old_start_date BETWEEN :first and :last", {first: range.first, last: range.last})
+  def self.project_step_date_filter(range, scope = ProjectStep, project_id)
+    scoped_steps = scope.where("actual_end_date BETWEEN :first AND :last OR scheduled_start_date BETWEEN :first AND :last "\
```

Comment:
Wrap these parameters so they don't go past 100 characters. Indent the second line of parameters.

````
scoped_steps = scope.where(.....
    .....)
````